src/Bundler.js

```diff
@@ -332,14 +335,28 @@ class Bundler extends EventEmitter {
       let thrown = err;
 
       if (thrown.message.indexOf(`Cannot find module '${dep.name}'`) === 0) {
+        let isLocalFile = dep.name.startsWith('.');
```

Comment:
Could the dependency be an absolute path, which would still be a local file? Trying to install in that scenario would blow up on `npm install`.

Reply:
Ya you’re right.

I would also have to add support for tilde paths etc now that #850 landed.

Reply:
You'll need something like let isLocalFile = /^[/~.]/.test(dep.name); in order to support the new resolver. See https://github.com/parcel-bundler/parcel/blob/master/src/Resolver.js#L88-L114